src/data-formatter.ts

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+import { FeatureInterface } from './feature';
+
+export interface FeaturesBase {
+    features: FeatureInterface[];
+}
+
+export interface Features extends FeaturesBase {
+    version: number;
+    features: FeatureInterface[];
+}
+
+export function toNewFormat (data: any) : Features {
+    return {
+        version: 1,
+        features: data.features.map((feature: any) : FeatureInterface => {
+            const copied: FeatureInterface = Object.assign({}, feature);
+            if (!feature.strategies && feature.strategy) {
+                copied.strategies = [{ name: feature.strategy, parameters: feature.parameters }];
+                return copied;
```

Comment:
feature.strategy is still set, but I guess it is not "available" because of the typescript interface?